README.md

```diff
@@ -1,2 +1,372 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: ocp-virt-validation-sa
+  namespace: ocp-virt-validation
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: ocp-virt-validation-cluster-admin-binding
+subjects:
+  - kind: ServiceAccount
+    name: ocp-virt-validation-sa
+    namespace: ocp-virt-validation
+roleRef:
+  kind: ClusterRole
+  name: cluster-admin
+  apiGroup: rbac.authorization.k8s.io
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: ocp-virt-validation-pvc-20250520-105358
+  namespace: ocp-virt-validation
+  labels:
+    app: ocp-virt-validation
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 1Gi
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: ocp-virt-validation-job-20250520-105358
+  namespace: ocp-virt-validation
+spec:
+  template:
+    metadata:
+      labels:
+        app: ocp-virt-validation
+    spec:
+      serviceAccountName: ocp-virt-validation-sa
+      securityContext:
+        fsGroup: 1001
+      containers:
+        - name: ocp-virt-validation-checkup
+          image: registry.redhat.io/container-native-virtualization/ocp-virt-validation-checkup-rhel9:v4.19.0-28
+          imagePullPolicy: Always
+          env:
+            - name: DRY_RUN
+              value: "false"
+            - name: TIMESTAMP
+              value: 20250520-105358
+            - name: RESULTS_DIR
+              value: /results
+            - name: TEST_SUITES
+              value: compute,network,storage,ssp
+            - name: TEST_SKIPS
+              value: ""
+          volumeMounts:
+            - name: results-volume
+              mountPath: /results
+      restartPolicy: Never
+      volumes:
+        - name: results-volume
+          persistentVolumeClaim:
+            claimName: ocp-virt-validation-pvc-20250520-105358
+  backoffLimit: 0
+```
+In order to apply these manifests directly onto the cluster in order to start the validation checkup, run:
+```bash
+podman run -e OCP_VIRT_VALIDATION_IMAGE=${OCP_VIRT_VALIDATION_IMAGE} ${OCP_VIRT_VALIDATION_IMAGE} generate | oc apply -f -
+```
+You can use output redirection to a file in order to make adjustments, if needed:
+```bash
+podman run -e OCP_VIRT_VALIDATION_IMAGE=${OCP_VIRT_VALIDATION_IMAGE} ${OCP_VIRT_VALIDATION_IMAGE} generate > run_manifests.yaml
+```
+
+### Make modifications
+The default settings of the validation checkup can be modified before the execution.
+
+#### Test Suites
+By default, the validation checkup will run all of the test suites.  
+It is possible to configure a subset of the suites to run. The available test suites are:
+* `compute`
+* `network`
+* `storage`
+* `ssp`
+
+Use the `TEST_SUITES` environment variable, with a comma separated list of the desired suites, when running the `generate` script. The example below, shows how to run only the `compute` and the `network` suites, by setting the `TEST_SUITES` to `"compute,network"`.Example:
```

Comment:
Please remove the last "Example" word, because the text already refer to "thd exame below", or at least add two new lines (to force th he markdown to add new line), before this word.

Reply:
thanks, done.